terracumber/terraformer.py

```diff
@@ -66,12 +66,12 @@ def inject_repos(self, custom_repositories_json):
                     node_mu_repos = repos.get(node, None)
                     replacement_list = ["additional_repos = {"]
                     for name, url in node_mu_repos.items():
-                        replacement_list.append(f'\n{" " * 4}{name} = "{url}"')
+                        replacement_list.append(f'\n    "{name}" = "{url}"')
                     replacement_list.append("\n  }")
                     replacement = ''.join(replacement_list)
-                    placeholder = '//' + node + '_additional_repos'
+                    placeholder = f'//{node}_additional_repos'
                     n_replaced = 0
-                    for line in fileinput.input("%s/main.tf" % self.terraform_path, inplace=True):
+                    for line in fileinput.input(f"{self.terraform_path}/main.tf", inplace=True):
```

Comment:
The string formatting across terracumber still uses the old Python 2.x style, so this change should be optional.

Reply:
I'm changing it to 3X when I update a method. #!/usr/bin/python3 is use for this script